alfresco-zipkin-repo/build.gradle

```diff
@@ -22,29 +24,18 @@ dependencies {
     testRuntime group: 'javax.servlet', name: 'servlet-api', version: '2.5'
 }
 
-amp {
-    from("$projectDir/src/main/resources"){
-        include '**/*.properties'
-        expand( artifactId: "$project.name",
-                title: "$project.name",
-                description: "$description",
-                version: "$project.version")
-    }
-    from("$projectDir/src/main/resources"){
-        exclude '**/*.properties'
+project.tasks.amp.config {
+    from files("${project.projectDir}/src/main/amp/") {
```

Comment:
If you moved the files to be packaged to `src/main/amp/config` (as the conventional layout prescribes), there would be no need to add this `from()`

Reply:
@vierbergenlars I preserved the AMP structure as it was. Moving it to config would change the location of the Solr configuration in the AMP, which I am not 100% sure where to place.
Still convinced we should move this?